spec/steps/household_add_member_spec.rb

```diff
@@ -10,4 +10,35 @@
   end
 
   include_examples "social security number"
+
+  context "relationship is included in list" do
+    it "is valid" do
+      valid_relationship = "Roommate"
+
+      step = HouseholdAddMember.new(relationship: valid_relationship)
+      step.valid?
+
+      expect(step.errors[:relationship]).to eq []
```

Comment:
I believe you can do `expect(step).to be_valid` also and it might be the same thing and you can avoid calling `.valid?`.

http://www.rubydoc.info/gems/rspec-rails/RSpec/Rails/Matchers#be_valid-instance_method

Reply:
ah yeah -- the thing is, this is not a valid step because there are presence validations on other fields :)